enhancements/installer/openstack-customer-provided-subnets.md

```diff
@@ -0,0 +1,131 @@
+---
+title: allow-customer-provisioned-aws-subnets
+authors:
+  - "@wking"
+reviewers:
+  - "@mandre"
+  - "@pierreprinetti"
+  - "@adduarte"
+approvers:
+  - "@abhinavdahiya"
+  - "@sdodson"
+creation-date: 2020-03-12
+last-updated: 2020-03-12
+status: planning
+---
+
+# 4.5: OpenStack: Allow Customer Provisioned Subnets
+
+## Release Signoff Checklist
+
+- [ x ] Enhancement is `implementable`
+- [ x ] Design details are appropriately documented from clear requirements
+- [ x ] Test plan is defined
+- [ x ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+In many large organizations, IT groups are separated into teams. A common separation of duties lies between networking/WAN and application or enabling technology deployments. One group will provision or assign OpenStack accounts with core management technology. A second group will be responsible for provisioning networks complete with subnet & zone selection, routing, VPN connections, and ingress/egress rules from the company. Lastly, an applications team (or individual business units) are given a quota for other OpenStack resources to provision application-specific items (instances, containers, databases, pipelines, load balancers, etc.).
+Another example of this split is found in companies that provide public access to their OpenStack resources, giving their tenants/customers quota on compute and storage resources but providing pre-defined networks for them.
+In these scenarios, only OpenStack administrators will manage these networks, which will be generally provider networks, (as opposed to tenant networks), since they require physical network infrastructure configuration (typically VLANs). So when OpenStack tenants log in they will find them pre-created, shared with them and ready to use.
+To accomplish this, our infrastructure needs an OpenStack account access that can be separated into "networking infrastructure" versus "application". Networking infrastructure includes elements related to the networks (networks, subnets, routing tables, internet gateways, NAT, VPN). Application items include OpenStack resources directly touching nodes within the cluster (LBs, security groups, Swift containers, nodes). This is a typical separation of responsibilities among IT groups and deployment/provisioning pipelines within large companies.
```

Comment:
a separating line between paragraphs can help separate the line of though here from _how companies setup_ to what we want to deliver to our customers..

Reply:
Line 33 reads as we will be requesting 2 accounts, one for "networking infrastructure" one for "application". I do not believe this is the intention as this implies being given a cluster admin account (very unlikely) to create the external networking configuration in OpenStack. 

Consider:
"...an OpenStack account with access to the pre-provisioned  networking infrastructure...."